src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionStringBuilder.cs

```diff
@@ -169,1766 +176,1672 @@ static SqlConnectionStringBuilder()
+            public override bool GetStandardValuesExclusive(ITypeDescriptorContext context) => false;
+
+            public override StandardValuesCollection GetStandardValues(ITypeDescriptorContext context)
             {
-                if (value < 0)
+                // There can only be standard values if the connection string is in a state that might
+                // be able to instantiate a connection
+                if (GetStandardValuesSupportedInternal(context))
                 {
-                    throw ADP.InvalidConnectionOptionValue(DbConnectionStringKeywords.ConnectTimeout);
+                    // Create an array list to store the database names
+                    List<string> values = new List<string>();
+
+                    try
+                    {
+                        SqlConnectionStringBuilder constr = (SqlConnectionStringBuilder)context.Instance;
+
+                        // Create a connection
+                        using (SqlConnection connection = new SqlConnection())
+                        {
+                            // Create a basic connection string from current property values
+                            connection.ConnectionString = constr.ConnectionString;
+
+                            // Try to open the connection
+                            connection.Open();
+
+                            DataTable databaseTable = connection.GetSchema("DATABASES");
+
+                            foreach (DataRow row in databaseTable.Rows)
+                            {
+                                string dbName = (string)row["database_name"];
+                                values.Add(dbName);
+                            }
+                        }
+                    }
+                    catch (SqlException e)
+                    {
+                        ADP.TraceExceptionWithoutRethrow(e);
+                        // silently fail
+                    }
+
+                    // Return values as a StandardValuesCollection
+                    return new StandardValuesCollection(values);
                 }
-                SetValue(DbConnectionStringKeywords.ConnectTimeout, value);
-                _connectTimeout = value;
+                return null;
             }
         }
 
-        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/CurrentLanguage/*' />
-        [DisplayName(DbConnectionStringKeywords.CurrentLanguage)]
-        [ResCategoryAttribute(StringsHelper.ResourceNames.DataCategory_Initialization)]
-        [ResDescriptionAttribute(StringsHelper.ResourceNames.DbConnectionString_CurrentLanguage)]
-        [RefreshPropertiesAttribute(RefreshProperties.All)]
-        public string CurrentLanguage
+        internal sealed class SqlConnectionStringBuilderConverter : ExpandableObjectConverter
```

Comment:
This class isn't used in netcore.

Reply:
It may not be used directly but if it's ever referenced it can be created through the component provider infrastructure. It's probably best to keep it around.

Reply:
It wasn't but I've included it in netcore to narrow the diverge.